.eslintignore

```diff
@@ -0,0 +1 @@
+test/async-function-tests.js
```

Comment:
Please create an issue for this and ping me-- ESLint definitely supports parsing of async function and I'm happy to take care of it when I get some free time.